src/config/link.rs

```diff
@@ -0,0 +1,41 @@
+use std::fmt::Display;
+
+use serde::{Deserialize, Serialize};
+
+use super::super::is_default;
+
+#[derive(Default, Serialize, Deserialize, PartialEq, Eq, Debug, Clone, schemars::JsonSchema)]
+pub enum LinkType {
+    #[default]
+    Config,
+    Protobuf,
+}
+
+impl Display for LinkType {
```

Comment:
This we can derive, can't we?

Reply:
it does not work for enum, optionally we can install an external crate like [enum_display_derive](https://docs.rs/enum-display-derive/latest/enum_display_derive/) or [strum](https://docs.rs/strum/0.26.1/strum/)